src/main/resources/db/changelog/changes/01-create-books-table.yaml

```diff
@@ -36,13 +36,13 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: description
-                  type: varchar(1024)
+                  type: varchar(1028)

```

Comment:
Why did you change it?

Reply:
I probably made some changes, when was implementing validation. But then figured out it was wrong decision, so I reversed changes and dropped db at that time. Although it seems I reversed this one incorrectly, so now it remains with current DB.